### ISSUE TITLE
Zentimings states that memory is SR but motherboard manufacturers list it as DR

### ISSUE THREAD

**Issue by Nostras** [irusanov/ZenTimings#35]
As title states. I have some Kingston DDR5 memory 2x32GB that Zentimings says is SR, but all motherboard menufacturers say DR.
![image](https://user-images.githubusercontent.com/4203238/215285671-2ed829a3-e8f7-47ef-a684-a38631b3718b.png)
![image](https://user-images.githubusercontent.com/4203238/215285754-787cbea9-ee09-407b-af84-85fe11a26dbf.png)

You can check the memory below:
https://www.msi.com/Motherboard/MEG-X670E-ACE/support#mem
https://rog.asus.com/motherboards/rog-crosshair/rog-crosshair-x670e-hero-model/helpdesk_qvl_memory/
https://www.gigabyte.com/Motherboard/X670E-AORUS-MASTER-rev-10/support#support-memsup

As 2x 32GB is DR as far as I can tell this seems like an oops in Zentimings.

**Comment by irusanov** [comment]
@Nostras Yes, this is a known issue (basically SR/DR is completely broken for DDR5). I'm working on it.

**Comment by irusanov** [comment]
Should be fixed with more recent version.
